src/core/lv2/Lv2Ports.cpp

```diff
@@ -126,75 +127,106 @@ std::vector<PluginIssue> Meta::get(const LilvPlugin *plugin,
 		issue(unknownPortFlow, portName);
 	}
 
-	m_def = .0f; m_min = .0f; m_max = .0f;
+	m_def = .0f;
+	m_min = std::numeric_limits<decltype(m_min)>::lowest();
+	m_max = std::numeric_limits<decltype(m_max)>::max();
```

Comment:
@JohannesLorenz This is a problem because later you perform `if (m_max - m_min > 15.0f)` which leads to an overflow. Sorry, I seem to have had FE_OVERFLOW commented out while testing this PR during the review.

```
Thread 1 "lmms" received signal SIGFPE, Arithmetic exception.
Lv2Ports::Meta::get (this=0x7fffffffd8c0, plugin=0x5555561edf50, portNum=11) at /home/zonkmachine/builds/lmms/src/core/lv2/Lv2Ports.cpp:221
221				if (m_max - m_min > 15.0f)
```

Reply:
Ouch! I forgot a test review after the code review! Sorry. I'll open a further PR and add you on that PR, hope that's OK.

Reply:
NP. On the plus side, now I know what the FE_OVERFLOW flag is used for. :)

Reply:
@zonkmachine Remember to write down your knowledge into the wiki, if appropriate.

Reply:
Resolving, as this is continued in #5811